common/src/chain/block/consensus_data.rs

```diff
@@ -105,4 +114,15 @@ impl PoWData {
     pub fn update_nonce(&mut self, nonce: u128) {
         self.nonce = nonce;
     }
+
+    pub fn get_block_proof(&self) -> Option<Uint256> {
```

Comment:
should these be without the `get_` prefix?

Reply:
get_ is OK if there's a calculation involved underneath.